bindings/wasm/src/iota/iota_did.rs

```diff
@@ -136,6 +137,12 @@ impl WasmIotaDID {
     WasmIotaDIDUrl::from(self.0.to_url())
   }
 
+  /// Creates an AliasId from the DID tag.
+  #[wasm_bindgen(js_name = toAliasId)]
+  pub fn to_alias_id(&self) -> String {
+    Into::<AliasId>::into(&self.0).to_string()
+  }
```

Comment:
Maybe mention the string encoding for clarity.

```suggestion
  /// Extracts the hex-encoded AliasId with a '0x' prefix, from the DID tag.
  #[wasm_bindgen(js_name = toAliasId)]
  pub fn to_alias_id(&self) -> String {
    AliasId::from(&self.0).to_string()
  }
```